<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamAndroid</class>
 <widget class="qMRMLWidget" name="StreamAndroid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>503</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="configCollapsibleButton" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text" stdset="0">
      <string>Configuration</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input Device</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="deviceComboBox"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Bitrate</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="bitrateSpinBox">
        <property name="maximum">
         <number>800000000</number>
        </property>
        <property name="value">
         <number>8000000</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Max FPS</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="fpsSpinBox">
        <property name="maximum">
         <number>40</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Max Width</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QSpinBox" name="maxWidthSpinBox">
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="value">
         <number>1080</number>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="flipImageCheckBox">
          <property name="text">
           <string>Flip Image</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="stayAwakeCheckBox">
          <property name="text">
           <string>Stay Awake</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="refreshDeviceListButton">
          <property name="text">
           <string>Refresh Device List</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Output Volume</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="qMRMLNodeComboBox" name="outputVolumeComboBox" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="nodeTypes" stdset="0">
         <stringlist notr="true">
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="hideChildNodeTypes" stdset="0">
         <stringlist notr="true"/>
        </property>
        <property name="noneEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="editEnabled" stdset="0">
         <bool>true</bool>
        </property>
        <property name="renameEnabled" stdset="0">
         <bool>true</bool>
        </property>
        <property name="interactionNodeSingletonTag" stdset="0">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="outputsCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Streaming</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="startStreamingButton">
          <property name="text">
           <string>Start Streaming</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopStreamingButton">
          <property name="text">
           <string>Stop Streaming</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
